test/autoschedulers/mullapudi2016/reorder.cpp

```diff
@@ -31,7 +32,7 @@ double run_test_1(bool auto_schedule) {
         // Provide estimates on the pipeline output
         r.set_estimates({{0, W}, {0, H}, {0, 3}});
         // Auto-schedule the pipeline
-        p.apply_autoscheduler(target, {"Mullapudi2016"});
+        p.apply_autoscheduler(target, get_autoscheduler_params(target.has_gpu_feature()));
```

Comment:
Write a GPU schedule below in the manual scheduling case as well. Otherwise we compare GPU autoschedule against CPU manual schedule.

Reply:
It's not really a matter of performance benchmarking. The tests simply check if the autoscheduler doesn't do something wildly wrong, by allowing a 2x performance hit comapred to the manual schedule. If it's slower than half the speed, it's considered a correctness issue.

If you already have schedules available for a follow-up PR, which you feel are out-of-scope for this PR, I guess we can solve this next PR.